packages/editor/src/components/blocks/button/Button.tsx

```diff
@@ -89,7 +100,39 @@ export const useButtonComponent = () => {
           type: 'generic',
           render: renderIconField
         },
+
         processOnlySelf: { subsection: 'Behaviour', type: 'hidden' },
+        confirmDialog: {
+          section: 'Confirm Dialog',
+          subsection: 'General',
+          label: t('components.button.property.confirmDialog'),
+          type: 'checkbox',
+          hide: data => data.type !== 'DELETE'
+        },
+        cdSeverity: {
```

Comment:
I would just work with the term "confirm".
I think I wouldn't be aware what you mean here with cd, so I would rename this here to confirm... e.g. confirmSeverity.